homeassistant/components/homematicip_cloud/__init__.py

```diff
@@ -86,6 +90,13 @@
     {vol.Optional(ATTR_ACCESSPOINT_ID): vol.All(str, vol.Length(min=24, max=24))}
 )
 
+SCHEMA_SET_ACTIVE_CLIMATE_PROFILE = vol.Schema(
+    {
+        vol.Optional(ATTR_ENTITY_ID): comp_entity_ids,
```

Comment:
`entity_id` should be required according to our new standard. Use keyword `"all"` to target all entities.

Reply:
done